Packs/Cybersixgill-ActionableAlerts/Layouts/layoutscontainer-Cybersixgill_Actionable_Alerts.json

```diff
@@ -0,0 +1,587 @@
+                            }
+                        ],
+                        "maxW": 3,
+                        "minH": 1,
+                        "minW": 1,
+                        "moved": false,
+                        "name": "Closing Information",
+                        "static": false,
+                        "w": 1,
+                        "x": 2,
+                        "y": 4
+                    },
+                    {
+                        "displayType": "CARD",
+                        "h": 2,
+                        "hideName": false,
+                        "i": "1vduzkpmlh-d8316060-ac70-11e9-a30b-53d47e1ea7d7",
+                        "items": [
+                            {
+                                "dropEffect": "move",
+                                "endCol": 2,
+                                "fieldId": "cybersixgillportalurl",
+                                "height": 53,
+                                "id": "d010b1f0-46b6-11ec-b9ff-23a663174c20",
+                                "index": 0,
+                                "listId": "1vduzkpmlh-d8316060-ac70-11e9-a30b-53d47e1ea7d7",
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            },
+                            {
+                                "endCol": 1,
+                                "fieldId": "cybersixgillstatus",
+                                "height": 53,
+                                "id": "3bf8dc80-4617-11ec-a3e6-073e7f7f29fa",
+                                "index": 1,
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            },
+                            {
+                                "endCol": 2,
+                                "fieldId": "cybersixgillalertid",
+                                "height": 53,
+                                "id": "3abd4f40-4617-11ec-a3e6-073e7f7f29fa",
+                                "index": 1,
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            },
+                            {
+                                "dropEffect": "move",
+                                "endCol": 1,
+                                "fieldId": "cybersixgilltriggeredassets",
+                                "height": 53,
+                                "id": "96c217b0-46ff-11ec-a87f-9f90967a78ae",
+                                "index": 1,
+                                "listId": "1vduzkpmlh-d8316060-ac70-11e9-a30b-53d47e1ea7d7",
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            },
+                            {
+                                "endCol": 2,
+                                "fieldId": "xdrseverity",
```

Comment:
Using XDR field will cause dependence, is it intentional?

Reply:
I created the layout with the UI and took the json from there, can you explain the meaning of the dependence?

Reply:
By using fields from other content packs such as XDR you will create dependence, that means that when customer wants to download your pack he will have to download XDR pack as well due to use of content artifacts from other packs.
Just remove from your JSON fields that you don't use in your layout.